stylix/palette.nix

```
@@ -159,9 +160,9 @@ in
     # https://github.com/SenchoPens/base16.nix/blob/b390e87cd404e65ab4d786666351f1292e89162a/README.md#theme-step-22
     lib.stylix.colors = (cfg.base16.mkSchemeAttrs cfg.base16Scheme).override cfg.override;
 
-    assertions = [
+    assertions = lib.mkIf cfg.enable [
       {
-        assertion = !(cfg.image == null && cfg.base16Scheme == null);
+        assertion = cfg.base16Scheme != null;
```

On second thought, it might be better to keep the previous predicate to avoid the implicit `stylix.image` dependency of `stylix.base16Scheme`:

```suggestion
        assertion = !(cfg.image == null && cfg.base16Scheme == null);
```